qdrant-landing/content/documentation/concepts/explore.md

```diff
@@ -186,6 +186,62 @@ client.recommend("{collection_name}", {
 
 Parameter `using` specifies which stored vectors to use for the recommendation.
 
+### Lookup vectors from another collection
+
+*Available as of v0.11.6*
+
+If you have collections with vectors of the same dimensionality,
+and you want to look for recommendations in one collection based on the vectors of another collection,
+you can use the `lookup_from` parameter.
+
+It might be useful, e.g. in the item-to-user recommendations where user and item embeddings are stored in different collections with different index configuration.
```

Comment:
It might be confusing to mention different index configuration, I think.

```suggestion
It might be useful, e.g. in the item-to-user recommendations where user and item embeddings are stored in different collections, but are still comparable and have the same dimensions.
```